README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01: Self-Introduction
+My name is **Xuefei Gao** and you can just call me **Ian**, which is easier to pronounce and remember. I am a raising second year and I plan to major in Math and Econ. I am also a fellow of SISRM and I am going to work with Professor Susan Stokes this summer.
+
+![Photo](Self_Portrait.png)
+
+Here are some of my interests:
+1. Music
+  * Rock
+  * Classical
+2. Basketball
+3. Sneakers
```

Comment:
Sneakers really are the best! Are you into the retro or more modern ones?

Reply:
I would say both. But recently I'm really into Converse. It's like going back to the good old days. :)